packages/core/src/lib/features/FeatureManager.ts

```diff
@@ -49,6 +52,10 @@ export class FeatureManager {
       this.addFeature(new AppsFlyerFeature(twaManifest.features.appsFlyer));
     }
 
+    if (twaManifest.features.firstRunFlag !== undefined) {
```

Comment:
I know this isn't the first PR with that pattern, but I feel a little weird that we're using an empty object to signify we should turn the feature on.

Reply:
We could add a `enabled: boolean` to the config. The test would then become:

```ts
if (twaManifest.features.firstRunFlag && twaManifest.features.firstRunFlag.enabled)  {
```

Reply:
Yeah, that works well for me.